apps/aecore/lib/aecore/peers/worker.ex

```diff
@@ -62,7 +62,7 @@ defmodule Aecore.Peers.Worker do
   def genesis_block_header_hash() do
     Block.genesis_block().header
     |> BlockValidation.block_header_hash()
-    |> Base.encode16()
+    |> Header.bech32_encode()
```

Comment:
For internal use I would leave binary representation.

Reply:
We use it to compare the genesis hash we get from client info, I think this is fine.

Reply:
better to compare using binary, as we use binary representation internally